tests/test_edgeql_expr_aliases.py

```diff
@@ -1304,6 +1304,437 @@ async def test_edgeql_aliases_schema_types_03(self):
             []
         )
 
+    async def _check_ddl_type_changes(self, ddls_and_expected_types):
```

Comment:
Can you test:
 * Create non-alias tuple reference
 * Create alias tuple reference
 * Drop non-alias tuple reference
 * Drop alias tuple reference
 * Type should not exist

I *think* this case is not handled, but I could be wrong?